lib/constants/networkRequest.js

```diff
@@ -1,21 +1,23 @@
+/* eslint-disable */
+
 const NETWORK_PROP = {
-	BODY: Symbol('body'),
-	METHOD: Symbol('method'),
-	STATUS: Symbol('status'),
+	BODY:   '@body',
```

Comment:
This should stay symbols as well. "@" is not a legal char for header, however I can imaging non-standard implementations where it's allowed.